### ISSUE TITLE
How to throttle down the CPU and GPU mining?

### ISSUE THREAD

**Issue by chakmeshma** [fireice-uk/xmr-stak#2140]
How is it possible to throttle down the execution and performance occupation impact of the program in run-time? What's the best way?

**Comment by Spudz76** [comment]
Linux offers `renice` or `nice` for which you can run at the lowest scheduling priority (19)

Windows you can pop Task Manager, find xmr-stak, and set process to low priority.

Both will run 100% when nothing else is using the CPU, but as soon as any other process wants CPU it will kick miner to the end of the queue.

Which GPU backend?  Nvidia has `bsleep` which you can raise to 100 or more which pauses every round for that many milliseconds (to let other GPU work have a slice).

**Comment by x151973** [comment]
> 
> 
> Linux offers `renice` or `nice` for which you can run at the lowest scheduling priority (19)
> 
> Windows you can pop Task Manager, find xmr-stak, and set process to low priority.
> 
> Both will run 100% when nothing else is using the CPU, but as soon as any other process wants CPU it will kick miner to the end of the queue.
> 
> Which GPU backend? Nvidia has `bsleep` which you can raise to 100 or more which pauses every round for that many milliseconds (to let other GPU work have a slice).

Is there any setting like bsleep on AMD gpu?

**Comment by Spudz76** [comment]
No, but you can reduce the intensity which makes the loop come back more often / similar but without sleeping however it does still introduce more chances/timeslices for other things to process (system interactivity)

Obviously hurts hashrate
